src/componenets/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { Sling as Hamburger } from 'hamburger-react';
+import { AiOutlineBell } from 'react-icons/ai';
+import { RiArrowDownSLine } from 'react-icons/ri';
+import { BiSearch } from 'react-icons/bi';
+import profilePhoto from '../assets/profile-photo.jpeg';
+
+const Navbar = () => {
+  const [isOpen, setOpen] = useState(false);
+
+  return (
+    <div className="navbar">
+      <div className="hamburger">
+        <Hamburger toggled={isOpen} toggle={setOpen} size={20} rounded color="#7e7d7f" />
+        <nav className={isOpen ? 'navlinks Open' : 'navlinks'}>
+          <div className="mob-profile">
+            <img src={profilePhoto} alt="profile" className="profile-photo mob-profile-photo" />
+            <p className="username mob-username">Username</p>
+            <p>example@email.com</p>
+          </div>
+          <NavLink to="/" activeclassname="active" onClick={() => setOpen(false)}>Houses</NavLink>
```

Comment:
Use consistent naming conventions throughout your codebase to make it easier to read and maintain. For example, you could choose between 'activeclassname' or 'activeClassName' and stick to it.